dune_client/models.py

```diff
@@ -85,14 +89,18 @@ class ExecutionStatusResponse:
     query_id: int
     state: ExecutionState
     times: TimeData
+    # this will be present when the query execution completes
+    result_metadata: Optional[ResultMetadata]
```

Comment:
This is not present in ExecutionStatus Response Class.

https://github.com/cowprotocol/dune-client/issues/22